R/rdeephaven/R/aggregate_wrapper.R

```diff
@@ -1,95 +1,176 @@
-#' @export
 Aggregation <- R6Class("Aggregation",
   cloneable = FALSE,
   public = list(
     .internal_rcpp_object = NULL,
-    initialize = function(aggregation) {
-      if (class(aggregation) != "Rcpp_INTERNAL_Aggregate") {
-        stop("'aggregation' should be an internal Deephaven Aggregation. If you're seeing this,\n  you are trying to call the constructor of an Aggregation directly, which is not advised.\n  Please use one of the provided aggregation functions instead.")
+    .internal_num_cols = NULL,
+    initialize = function(aggregation, ...) {
+      args <- list(...)
+      if (any(names(args) == "cols")) {
+        self$.internal_num_cols <- length(args$cols)
       }
-      self$.internal_rcpp_object <- aggregation
+      self$.internal_rcpp_object <- do.call(aggregation, args)
     }
   )
 )
 
 ### All of the functions below return an instance of the above class
 
+#' @description
+#' Create a First aggregation that computes the group-wise first value
+#' of each column in `cols`, for each aggregation group.
+#' @param cols String or list of strings denoting the column(s) to aggregate. Can be renaming expressions, i.e. “new_col = col”.
+#' Default is to use all non-grouping columns, which is only valid in the `agg_all_by()` operation.
```

Comment:
Did not rip off.  Can someone with broken English as a 3rd language parse this?

Reply:
Pydoc is
`the column(s) to aggregate on, can be renaming expressions, i.e. “new_col = col”; default is None, only valid when used in Table agg_all_by operation`
I feel this is as close as I can accurately get to that, as now `default is None` has no meaning in R.